packages/storybook/stories/CallWithChatComposite/JoinExample.stories.tsx

```diff
@@ -34,12 +36,36 @@ const JoinTeamsMeetingStory = (args: ArgsFrom<typeof storyControls>, context): J
           endpointUrl: args.endpointUrl,
           locator: { meetingLink: args.teamsMeetingLink }
         });
+      }
+      if (
+        !!args.token &&
+        !!args.userId &&
+        !!args.endpointUrl &&
+        !!args.displayName &&
+        !!args.teamsMeetingId &&
+        !!args.teamsMeetingPasscode
+      ) {
+        setMeetingProps({
+          userId: { communicationUserId: args.userId },
+          token: args.token,
+          displayName: args.displayName,
+          endpointUrl: args.endpointUrl,
+          locator: { meetingId: args.teamsMeetingId, passcode: args.teamsMeetingPasscode }
+        });
       } else {
         setMeetingProps(undefined);
       }
     };
     fetchToken();
-  }, [args.token, args.userId, args.endpointUrl, args.displayName, args.teamsMeetingLink]);
+  }, [
+    args.token,
+    args.userId,
+    args.endpointUrl,
+    args.displayName,
+    args.teamsMeetingLink,
+    args.teamsMeetingId,
```

Comment:
I believe we had a discussion about simplifying the preview pages instead of adding everything on?

Reply:
Yes, spited them